source/core/assets/configs/player.json

```diff
@@ -9,5 +9,5 @@
   "textureFilename": "images/player/player.png",
   "textureAtlasFilename": "images/player/player.atlas",
   "melee": "melee:knife",
-  "ranged": "ranged:shotgun"
+  "ranged": "ranged:pistol"
```

Comment:
leave shotgun for debugging I will be changing this with the balance patch later today